apps/docs/src/app/core/component-docs/multi-input/examples/multi-input-form-example/multi-input-form-example.component.html

```diff
@@ -1,22 +1,26 @@
 <form [formGroup]="customForm">
-    <div fd-form-set class="flex-form-set">
+    <fieldset fd-fieldset class="flex-form-set">
         <div fd-form-item>
+            <label fd-form-label>Valid Localization Editor</label>
             <fd-multi-input
+                [state]="'valid'"
```

Comment:
Same here should not be constant valid but rather a function

Reply:
I added some functoin there.